Developing on Altis/private-packages.md

```diff
@@ -10,4 +10,6 @@ composer config --auth http-basic.my.yoast.com token {token}
  
 You should see a `auth.json` file in the project's directory - you can commit this, and it should authenticate correctly in the build. 
 
-`auth.json` is prevented from being accessed directly via the servers, so this won't be exposed publicly.
\ No newline at end of file
+`auth.json` is prevented from being accessed directly via the servers, so this won't be exposed publicly.
+
+[OWAIN, Should this mention secrets now we have them? - ie generate the auth.json as part of the build script tso it is not committed to the repo?]
```

Comment:
can remove this note. We can create a new issue to write a page on secrets!

Reply:
Just noting the auth.json case is explicitly part of the examples on https://docs.altis-dxp.com/cloud/build-scripts/private-dependencies/